grape-knock.gemspec

```diff
@@ -21,8 +21,10 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "grape", ">= 0.10"
   spec.add_dependency "knock"
+  spec.add_dependency "rails", ">= 4.2"
```

Comment:
i'm suprised we need the explicit dependency here since it's already declare in `knock` gem